nengo_spinnaker/builder/ensemble.py

```diff
@@ -34,6 +35,7 @@ def get_ensemble_sink(model, connection):
 
     if (isinstance(connection.pre_obj, nengo.Node) and
             not callable(connection.pre_obj.output) and
+            not isinstance(connection.pre_obj.output, Process) and
```

Comment:
What do you think of the test I just added? It definitely provides coverage of that code path, probably not so great at showing the input going into the communication channel is anything like white noise though...